packages/@aws-cdk/aws-appmesh/lib/shared-interfaces.ts

```diff
@@ -167,6 +167,31 @@ class FileAccessLog extends AccessLog {
   }
 }
 
+/**
+ * Represents the properties needed to define client policy
+ */
+export interface TlsClientPolicy {
+  /**
+   * Whether the policy is enforced.
+   *
+   * @default: true
+   */
+  readonly enforce?: boolean;
+
+  /**
+   * TLS is enforced on the ports specified here.
+   * If no ports are specified, TLS will be enforced on all the ports.
+   *
+   * @default: none
```

Comment:
1. No need for the `:` here.
2. The description of the default is not consistent with the description of the property. You probably want something like:

```suggestion
   * @default - TLS will be enforced on all ports
```

Reply:
Great point! I will simply this a bit and say "all ports".